subsys/net/ip/net_context.c

```diff
@@ -2458,6 +2458,29 @@ int net_context_update_recv_wnd(struct net_context *context,
 	return -EPROTOTYPE;
 #endif
 }
+
+bool net_context_is_bound(struct net_context *context)
+{
+	NET_ASSERT(context);
+
+#if defined(CONFIG_NET_IPV6)
```

Comment:
use IS_ENABLED with this CONFIG option, in the test right below.

Do the same for ipv4 also